app/src/main/java/org/p2p/wallet/history/ui/token/TokenHistoryPresenter.kt

```diff
@@ -40,6 +42,14 @@ class TokenHistoryPresenter(
         view?.openSellTransactionDetails(transactionId)
     }
 
+    override fun onReceiveClicked() {
+        if (ethAddressEnabledFeatureToggle.isFeatureEnabled) {
```

Comment:
you can extract this check to some ReceiveTokenFragmentFactory like we did in Swap
looks nice